resources/lang/en/general.php

```diff
@@ -1,244 +1,254 @@
-    'changeemail'  			=> 'Change Email Address',
-    'changepassword'  		=> 'Change Password',
-    'checkin'  				=> 'Checkin',
-    'checkin_from'  		=> 'Checkin from',
-    'checkout'  			=> 'Checkout',
+    'cancel'                => 'Cancel',
+    'categories'            => 'Categories',
+    'category'              => 'Category',
+    'change'                => 'In/Out',
+    'changeemail'           => 'Change Email Address',
+    'changepassword'        => 'Change Password',
+    'checkin'               => 'Checkin',
+    'checkin_from'          => 'Checkin from',
+    'checkout'              => 'Checkout',
     'checkouts_count'       => 'Checkouts',
     'checkins_count'        => 'Checkins',
     'user_requests_count'   => 'Requests',
-    'city'  				=> 'City',
-    'click_here'			=> 'Click here',
-    'clear_selection'		=> 'Clear Selection',
-    'companies'			    => 'Companies',
-    'company'				=> 'Company',
-    'component'			    => 'Component',
-    'components'			=> 'Components',
-    'complete'				=> 'Complete',
-    'consumable'			=> 'Consumable',
-    'consumables'			=> 'Consumables',
-    'country'  				=> 'Country',
-    'create'				=> 'Create New',
+    'city'                  => 'City',
+    'click_here'            => 'Click here',
+    'clear_selection'       => 'Clear Selection',
+    'companies'             => 'Companies',
+    'company'               => 'Company',
+    'component'             => 'Component',
+    'components'            => 'Components',
+    'complete'              => 'Complete',
+    'consumable'            => 'Consumable',
+    'consumables'           => 'Consumables',
+    'collapse'              => 'Collapse',
+    'count'                 => 'Count',
+    'country'               => 'Country',
+    'create'                => 'Create New',
     'created'               => 'Item Created',
-    'created_asset'			=> 'created asset',
-    'created_at' 			=> 'Created at',
-    'record_created' 		=> 'Record Created',
-    'updated_at' 			=> 'Updated at',
-    'currency'  			=> '$', // this is deprecated
-    'current'  				=> 'Current',
+    'created_asset'         => 'created asset',
+    'created_at'            => 'Created at',
+    'record_created'        => 'Record Created',
+    'updated_at'            => 'Updated at',
+    'currency'              => '$', // this is deprecated
+    'current'               => 'Current',
     'custom_report'         => 'Custom Asset Report',
-    'dashboard'				=> 'Dashboard',
-    'days'      			=> 'days',
+    'dashboard'             => 'Dashboard',
+    'dashboard_60_percent'  => '60 % Complete (warning)',
+    'dashboard_nothing_added_message' => 'It looks like you haven&apos;t added anything yet, so we don&apos;t have anything awesome to display. Get started by adding some assets, accessories, consumables, or licenses now!',
+    'dashboard_welcome_message' => 'This is your dashboard. There are many like it, but this one is yours.',
```

Comment:
`dashboard_60_percent`, `dashboard_nothing_added_message`, and `dashboard_welcome_message` are new phrases.